src/auro-accordion.js

```diff
@@ -100,24 +123,33 @@ export class AuroAccordion extends LitElement {
     content.style.height = `${height}px`;
   }
 
+  updated(changedProperties) {
+    if (changedProperties.has('iconRight')) {
+      this.requestUpdate();
+    }
+  }
+
   // function that renders the HTML and CSS into  the scope of the component
   render() {
 
-    const chevronHtml = this.chevron
-      ? html`
-          <div class="iconWrapper" part="chevron">
-            ${this.generateIconHtml(this.expanded ? chevronUp.svg : chevronDown.svg)}
-          </div>`
-      : html``
+    const chevronHtml =
```

Comment:
We need to preserve the ability to have a base trigger with no chevron. Convert the APi from `chevron` to `noChevron` and create a ticket in auro-sidenav top update it's template use of this after we release this update.